include/jemalloc/internal/hpa.h

```diff
@@ -147,6 +147,14 @@ struct hpa_shard_s {
 	 * Last time we performed purge on this shard.
 	 */
 	nstime_t last_purge;
+
+	/*
+	 * Last tick when we attempted work. If deferral work is allowed (we
+	 * have background thread), tick is the attempt of the background thread
+	 * to do the work.  If deferral is not allowed, tick is the time of
+	 * activity in the shard (hpa_alloc or hpa_dalloc).
```

Comment:
Is this used anywhere?  I couldn't find it by searching through your diff.

If it is, and we need to keep this, maybe start by saying "Last time we attempted to do work in the shard" since "tick" is jargon.  I'm not exactly sure what the "tick" is in the background case (is it time as well? or a count? or something else?) and since I don't see a use, maybe it could be clarified?